testgrid/cmd/entomologist/main_test.go

```diff
@@ -30,6 +32,18 @@ import (
 
 func TestOptions(t *testing.T) {
 
+	tmpdir, err := ioutil.TempDir("", "tmpdir")
```

Comment:
Similarly, if options don't need check to see if a file is _actually_ there, you might not need the complexity of faking files here.